examples/test.py

```diff
@@ -48,7 +48,7 @@
             print('GPU:  searchsorted in %0.3fms' % (1000*(time.time()-t0)))
 
             # compute the difference between both
-            error = torch.norm(test_CPU.to('cuda')
-                               - test_GPU).cpu().numpy()
+            error = torch.norm(test_CPU.to('cuda').double()
+                               - test_GPU.double()).cpu().numpy()
```

Comment:
this was made necessary for comparing the results, since `norm` doesn't accept long